.github/workflows/check-updates.yml

```diff
@@ -0,0 +1,20 @@
+name: Check if the Open API has changed
+on:
+  schedule:
+    - cron:  '0 */3 * * *'
```

Comment:
Every 3rd hour seems like a lot IMO, maybe once a day?

Reply:
I wanted to add an hour based scheduled so we could pick up changes on almost a same day window, but we can increase it so it doesn't run 8 times a day